src/approaches/interactive_learning_approach.py

```diff
@@ -391,7 +391,7 @@ def _query_policy(s: State) -> Optional[GroundAtomsHoldQuery]:
             atoms_to_query = set()
             for atom in ground_atoms:
                 score = self._score_atom_set({atom}, s)
-                if score > CFG.interactive_score_threshold:
+                if score > float(CFG.interactive_score_threshold):
```

Comment:
do we have a bug in our type checking for the CFG?

Reply:
nah this should be unnecessary, will take it out